gnupg/_parsers.py

```diff
@@ -74,6 +80,67 @@ def _check_keyserver(location):
                 return keyserver
             return None
 
+
+def _check_keyserver_option(ks_option):
+    """Check that the provided keyserver option is valid and safe.
+
+    :param str ks_option: A valid argument to --keyserver-option.
```

Comment:
It looks like you renamed the parameter to `option_value`?

Reply:
Oh, nope, I'm sorry. I appear to be used to C and Rust where the docstrings go _above_ the function they document. Carry on!

Reply:
Thanks, no worries!  I've solidly moved into Go these days so who knows what I'm going to write in during this merge :)